### ISSUE TITLE
Implement Post-Form Submission `atd-knack-api` Call for Inventory

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#1288]
We need to implement a `POST` request to the [`atd-knack-api`](https://github.com/cityofaustin/atd-knack-api) after any of the following form submissions in the mobile app:

- New Inventory Item
- Edit Inventory Item
- Cancel Inventory Item

The API request should fire once a `200` response is received from the form submission. It should take the following structure: `{base-url}?src={source-app-id}&dest={dest-app-id}`

No form data needs to be supplied to the API call.

### Base URL (base-url)

The URL endpoints are different for staging and production:
- Staging: `https://ywx4jkcwrh.execute-api.us-east-1.amazonaws.com/dev/inventory/`
- Production:`https://knack-api.austinmobility.io/inventory/`

### Source Application ID (src-app-id)

This is the application ID of the current Knack application (e.g. Data Tracker Production). This can be found in the `Knack` object under `Knack.application_id`.

### Destination Application ID (dest-app-id)

This is the Knack application ID of the Finance System. We currently only have a production instance of this app. Use `5b422c9b13774837e54ed814`.

We will provide a test application ID once our Knack development work is complete.

### Error handling

We do not currently have scheme for handling errors from this API call. I'll open a separate issue for this, once we decide on the best approach.

**Comment by mddilley** [comment]
@johnclary This is added to the mobile signals app. I created development and production configs to handle both environments, and this can be tested once the Knack development work mentioned above is complete.

**Comment by johnclary** [comment]
woohoo. thanks! will test soon

**Comment by johnclary** [comment]
@mddilley small change to make: would you update the base URL for staging to: `https://knack-api-dev.austinmobility.io/inventory` ?

Note that there is no trailing slash.

**Comment by mddilley** [comment]
@johnclary I just pushed this change, and the preview is finished building so you can test.

**Comment by johnclary** [comment]
@mddilley @mateoclarke i've tested this and it works beautifully. thanks!
